js/faq.js

```diff
@@ -1,4 +1,19 @@
 const faqTitles = document.querySelectorAll('.faq__title');
+const faqLinks = document.querySelectorAll('.faq_link');
+
+const currentLocation = window.location.hash;
+
+faqLinks.forEach((faqLink) => {
+  const faqLinkValue = faqLink.hash;
+  if (currentLocation === faqLinkValue) {
+    const ancTag = document.querySelector(`a[href="${faqLinkValue}"]`);
+    const faqNode = ancTag.parentNode.parentNode;
+    faqNode.classList.toggle('show');
+    const faqText = faqNode.nextElementSibling;
+    faqNode.childNodes[3].innerHTML = '-';
+    faqText.style.maxHeight = faqText.scrollHeight + 'px';
+  }
+});
```

Comment:
I see what you're trying to do here but since we already have the logic written for the case when a particular FAQ is clicked, what should be done. Can we somehow do something in such a way that it becomes that function's responsibility to handle the expanding concerned FAQ? Try to think from a user's point of view, after doing what is the FAQ being expanded and we're already handling the expanding and collapsing. Could we just trigger something such that the control goes to that logic and it figures out to what next to be done?

Reply:
I have written that logic inside a function. Please check.